docs/configuration/downstream/pull_from_upstream.md

```diff
@@ -9,8 +9,17 @@ A dist-git only job that opens a new dist-git pull request in
 src.fedoraproject.org when a new upstream release happens using a notification
 from [release-monitoring.org](https://release-monitoring.org/). Upstream Release Monitoring creates a Bugzilla
 for the new upstream release and this Bugzilla is by default referenced in the dist-git changelog as
-`- Resolves rhbz#xz` and `- Resolves rhbz#xz` in the commit message. This behaviour can be customized
-using [actions](/docs/configuration/actions#syncing-the-release).
+`- Resolves rhbz#xz` and `- Resolves rhbz#xz` in the commit message. 
+
+
+Bodhi updates created by the `bodhi_update` job as well as [automatic Bodhi updates](https://fedora-infra.github.io/bodhi/6.0/user/automatic_updates.html), for `rawhide`, will have this Bugzilla assigned and it will be closed when they reach stable.
```

Comment:
I wouldn't mention rawhide, because (as mentioned in the link) it's not the only branch automatic updates are created for, after branching happens there is a period of time when the branched release also receives automatic updates.